db/mongodb_test.go

```diff
@@ -76,4 +76,10 @@ func TestMongoDB(t *testing.T) {
 	if metadata2 != "forty-two" {
 		t.Errorf("expected foruty-two as the answer, got %s", metadata2)
 	}
+	if err := db.ExtraIndexes([]string{"teste.index1"}); err == nil {
+		t.Error("expected that this index would not be created.")
+	}
+	if err := db.ExtraIndexes([]string{"qsa.nome_socio"}); err != nil {
+		t.Errorf("expected the index to be created, got %s", err)
```

Comment:
Como [sugerido anteriorment](https://github.com/cuducos/minha-receita/pull/318#discussion_r2052864379):

```suggestion
		t.Errorf("expected no errors running extra indexes, got %s", err)
```